src/dependencies/dss-direct-deposit/D3MInit.sol

```diff
@@ -0,0 +1,326 @@
+}
+
+interface D3MAaveRateTargetPlanLike {
+    function rely(address) external;
+    function file(bytes32, uint256) external;
+    function adai() external view returns (address);
+    function stableDebt() external view returns (address);
+    function variableDebt() external view returns (address);
+    function tack() external view returns (address);
+    function adaiRevision() external view returns (uint256);
+}
+
+interface D3MAaveBufferPlanLike {
+    function file(bytes32, uint256) external;
+    function adai() external view returns (address);
+}
+
+interface ADaiLike {
+    function ATOKEN_REVISION() external view returns (uint256);
+}
+
+interface D3MCompoundPoolLike {
+    function hub() external view returns (address);
+    function dai() external view returns (address);
+    function ilk() external view returns (bytes32);
+    function vat() external view returns (address);
+    function file(bytes32, address) external;
+    function cDai() external view returns (address);
+    function comptroller() external view returns (address);
+    function comp() external view returns (address);
+}
+
+interface D3MCompoundRateTargetPlanLike {
+    function rely(address) external;
+    function file(bytes32, uint256) external;
+    function tacks(address) external view returns (uint256);
+    function delegates(address) external view returns (uint256);
+    function cDai() external view returns (address);
+}
+
+interface CDaiLike {
+    function interestRateModel() external view returns (address);
+    function implementation() external view returns (address);
+}
+
+interface D3MOracleLike {
+    function vat() external view returns (address);
+    function ilk() external view returns (bytes32);
+    function file(bytes32, address) external;
+}
+
+interface D3MHubLike {
+    function vat() external view returns (address);
+    function daiJoin() external view returns (address);
+    function file(bytes32, address) external;
+    function file(bytes32, bytes32, address) external;
+    function file(bytes32, bytes32, uint256) external;
+}
+
+interface D3MMomLike {
+    function setAuthority(address) external;
+}
+
+struct D3MCommonConfig {
+    address hub;
+    address mom;
+    bytes32 ilk;
+    bool existingIlk;
+    uint256 maxLine;
+    uint256 gap;
+    uint256 ttl;
+    uint256 tau;
+}
+
+struct D3MAavePoolConfig {
+    address king;
+    address adai;
+    address stableDebt;
+    address variableDebt;
+}
+
+struct D3MAaveRateTargetPlanConfig {
+    uint256 bar;
+    address adai;
+    address stableDebt;
+    address variableDebt;
+    address tack;
+    uint256 adaiRevision;
+}
+
+struct D3MAaveBufferPlanConfig {
+    uint256 buffer;
+    address adai;
+}
+
+struct D3MCompoundPoolConfig {
+    address king;
+    address cdai;
+    address comptroller;
+    address comp;
+}
+
+struct D3MCompoundRateTargetPlanConfig {
+    uint256 barb;
+    address cdai;
+    address tack;
+    address delegate;
+}
```

Comment:
Compound is not being used in this spell, so if the functions referencing these structs can be removed, so can them.